docs/source/model_doc/canine.rst

```diff
@@ -48,6 +48,8 @@ Tips:
   (which has a predefined Unicode code point). For token classification tasks however, the downsampled sequence of
   tokens needs to be upsampled again to match the length of the original character sequence (which is 2048). The
   details for this can be found in the paper.
+- Models: CANINE-C (~500 MB): Pre-trained with autoregressive character loss, 12-layer, 768-hidden, 12-heads, 121M
+  parameters. CANINE-S (~500 MB): Pre-trained with subword loss, 12-layer, 768-hidden, 12-heads, 121M parameters.
```

Comment:
```suggestion
  Models:
      - CANINE-C (~500 MB): Pre-trained with autoregressive character loss, 12-layer, 768-hidden, 12-heads, 121M parameters.
      - CANINE-S (~500 MB): Pre-trained with subword loss, 12-layer, 768-hidden, 12-heads, 121M parameters.
```
Oh I get why the script was unhappy, you had not properly formatted your list! Might need another make style after this suggestion is accepted, but it will keep the items in a proper list.